### PR TITLE
Update and rename QiDian.xml to QiDian.com.xml

### PR DESCRIPTION
**Pull request by ivysrono**
#10406 
#10538

### CODE REVIEW

**Comment by ivysrono** [comment]
#9906

File: src/chrome/content/rules/QiDian.com.xml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+		se.qidian.com
+		tiyu.qidian.com
+		wuxia.qidian.com
+		xianxia.qidian.com
+		xuanhuan.qidian.com
+		youxi.qidian.com
+		zhichang.qidian.com
+
+	Invalid certificate:
+		^qidian.com	( equal to www.qidian.com see details in QiDian-Problematic.xml)
```
Comment (J0WI) [inline src/chrome/content/rules/QiDian.com.xml]: You just removed that file

File: src/chrome/content/rules/QiDian.com.xml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+	MCB:
+		activity.qidian.com	http://activity.qidian.com/barrage
+
+	Redirected to http:
+		2cy.qidian.com
+		a.qidian.com
+		dushi.qidian.com
+		f.qidian.com
+		in.qidian.com
```
Comment (J0WI) [inline src/chrome/content/rules/QiDian.com.xml]: Should be `fin.qidian.com`

**Comment by J0WI** [comment]
Please do the rename in a separate commit next time, that makes it easier to read the diff.

**Pull request merged**